lib/features/caching/caching_manager.dart

```diff
@@ -118,6 +118,15 @@ class CachingManager {
     ], eagerError: true);
   }
 
+  Future<void> clearEmailCacheAndAllStateCacheAndMailboxCache() {
+    return Future.wait([
+      _stateCacheClient.clearAllData(),
+      _emailCacheClient.clearAllData(),
+      _mailboxCacheClient.clearAllData(),
+      if (PlatformInfo.isMobile) clearAllFileInStorage(),
+    ], eagerError: true);
+  }
```

Comment:
IMO, we don't need remove data cache

Reply:
he upgrade model for Mailbox, so I request to migrate data. Why we dont need?

Reply:
So we just only remove mailbox cache.